docs/CHANGELOG.md

```diff
@@ -32,12 +35,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### April 9, 2019
 - The last of the compiler flags are figured out
 
+### March 22, 2019
+- Deviliution appears on [Phoronix](https://www.phoronix.com/scan.php?page=news_item&px=DeviluitionX-Open-Diablo)
```

Comment:
```suggestion
- Devilution appears on [Phoronix](https://www.phoronix.com/scan.php?page=news_item&px=DeviluitionX-Open-Diablo)
```

Reply:
Oh the irony, I also spelled it wrong when I sent them the pitch :D

Reply:
Deviliution actually sounded pretty cool ;)